pkgs/by-name/mc/mcat/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "mcat";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "Skardyy";
+    repo = "mcat";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+wpcLf94FJhkufytKgVnRonivH9CVY/X0+bXx3Z8HKI=";
+  };
+
+  cargoHash = "sha256-pi/nts8YqXms5ps6bt+KQRh8twvk5qfkh5FF+/0tMOE=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "sha256-pi/nts8YqXms5ps6bt+KQRh8twvk5qfkh5FF+/0tMOE=";
```
(only required if you wanted to backport this to 24.11)